src/SprykerEco/Zed/Computop/Business/Payment/Reader/ComputopPaymentReader.php

```diff
@@ -50,9 +50,10 @@ public function isComputopPaymentExist(QuoteTransfer $quoteTransfer): QuoteTrans
      */
     protected function getPaymentTransfer(QuoteTransfer $quoteTransfer): TransferInterface
     {
-        $paymentSelection = $quoteTransfer->getPayment()->getPaymentSelection();
+        $paymentTransfer = $quoteTransfer->getPaymentOrFail();
+        $paymentSelection = $paymentTransfer->getPaymentSelectionOrFail();
```

Comment:
please remoive orFail here
it was not required before

Reply:
Fixed

Reply:
Those are not breaking changes if the expectation was for them to never be null - in other words: If it fails without them being set/present!